### DISCUSSION TITLE
How can I add servers dynamically in plugin?

### DISCUSSION THREAD

**Discussion by Harbon** [WaterdogPE/WaterdogPE#139] [Q&A]
I noticed that getProxy().registerServerInfo can add server dynamically , but it not working,   server is added,  connection always timeout. However when I add server in config.yml , connect successfully.

here is my plugin code
 ` ServerInfo serverInfo = this.getProxy().getServerInfoMap().createServerInfo("test123", InetSocketAddress.createUnresolved("kubernetes.docker.internal", 25565), InetSocketAddress.createUnresolved("kubernetes.docker.internal", 25565), ServerInfoMap.DEFAULT_TYPE);
        this.getProxy().registerServerInfo(serverInfo);`

**Comment by belohnung** [comment]
Are you sure port 25565 is the right port? Bedrock Edition uses 19132 per default.
